### PR TITLE
fix parse pd-urls in binlogctl

### PR DESCRIPTION
**Pull request by july2993**
### What problem does this PR solve? 
fix https://github.com/pingcap/tidb-binlog/issues/923
fail since #750 
### What is changed and how it works?
just copy code from master in `pkg/flag`

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

Side effects

Related changes
 - Need to be included in the release note

### CODE REVIEW

File: pkg/flags/urls.go [hunk 1]
```diff
@@ -64,17 +77,35 @@ func NewURLsValue(init string) (*URLsValue, error) {
...
-			return nil, errors.Trace(err)
+			return nil, errors.Errorf("parse url %s failed %v", str, err)
 		}
-		addrs = append(addrs, str)
+		if u.Scheme != "http" && u.Scheme != "https" && u.Scheme != "unix" && u.Scheme != "unixs" {
+			return nil, errors.Errorf("URL scheme must be http, https, unix, or unixs: %s", str)
+		}
+		if _, _, err := net.SplitHostPort(u.Host); err != nil {
+			return nil, errors.Errorf(`URL address does not have the form "host:port": %s`, str)
+		}
```
Comment (kennytm) [inline pkg/flags/urls.go]: unix??

Even if you're going to support unix-domain sockets, the `unix:///tmp/etcd.sock` will have an empty `u.Host` and cause `net.SplitHostPort` to fail ("missing port in address").

Comment (july2993) [reply pkg/flags/urls.go]: I don'ts what it wants actually, but after change changing to using `flags.NewURLsValue`
the endpoings get here will be int the formort:  `unix://localhost:4085651753091690150`
https://github.com/pingcap/tidb-binlog/blob/649ec60f9ceb65fbbaf22e06e43e39ae62e56df9/pump/node_test.go#L58

there's some similar code it wil use by `flag.NewUrLsValue`...
https://github.com/pingcap/tidb-binlog/blob/649ec60f9ceb65fbbaf22e06e43e39ae62e56df9/pkg/types/urls.go#L40

**Comment by july2993** [comment]
close for inactive

**Pull request closed without merging**